pkg/smart/doc.go

```diff
@@ -55,29 +55,30 @@ An example usage can be like this -
 package smartusageexample
 
 import (
+
 	"fmt"
 
-	"k8s.io/klog"
+	"k8s.io/klog/v2"
 	"github.com/openebs/node-disk-manager/pkg/smart"
+
 )
 
-func main() {
-	deviceBasicSCSIInfo, err := smart.SCSIBasicDiskInfo("/dev/sda")
-	if err != nil {
-		klog.Fatal(err)
-	}
+	func main() {
```

Comment:
Why is this formatting needed?

Reply:
It's the IDE. I'll fix this

Reply:
I think I got all of it. Comment formatting reverted.